Dockerfile

```diff
@@ -4,6 +4,7 @@ WORKDIR /airbyte/integration_code
 
 # Copy project files needed for build
 COPY pyproject.toml poetry.lock README.md ./
+COPY dist/*.whl ./dist/
```

Comment:
Copy the packages obtained from the build

Reply:
A comment + link to code on what builds it and why we think it'll be there would help!